remotespark/kernels/kernelmagics.py

```diff
@@ -163,6 +190,7 @@ def spark(self, line, cell="", local_ns=None):
     @argument("-n", "--maxrows", type=int, default=None, help="Maximum number of rows that will be pulled back "
                                                                         "from the server for SQL queries")
     @argument("-r", "--samplefraction", type=float, default=None, help="Sample fraction for sampling from SQL queries")
+    @_event
```

Comment:
Most of the magics are trivial and don't really have much functionality outside of argument parsing and calling a single method on the spark controller.  I've chosen to give them all an event because it doesn't really give the code any more complexity to do so, and it prevents us from having to add it later if any of the magics get more complicated.